.github/ISSUE_TEMPLATE/plugin_template.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 'Plugin suggestion'
+about: 'Suggest a new Backstage plugin'
+title: '[Plugin] THE PLUGIN NAME'
+labels: plugin
+---
+
+<!--- Thanks for suggesting a new plugin! -->
+<!--- If you plan to work on this plugin, please Assign yourself or your team member(s). If not, please add the 'help wanted' label -->
```

Comment:
Most ppl reporting issues are not allowed to add labels, maybe ask to mention in the issue instead?

Reply:
Good point @Rugvip -- do you think they are allowed to assign themselves?

Reply:
@stefanalund I can not assign myself to any issue. Only the `maintainers` of the repository can assign